package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "com.mcpiroman.unity-native-tool",
+  "displayName": "Unity Native Tool",
+  "version": "7.1.0",
+  "description": "Tool for unloading native DLLs in the editor. \nhttps://github.com/mcpiroman/UnityNativeTool",
+  "keywords": [
+    "native",
+    "c++",
+    "dll"
+  ],
+  "category": "Unity",
+  "author": {
+    "name": "John Doe",
```

Comment:
If I recall correctly, that's not my name :)

Reply:
Nor an email

Reply:
Where can I find them? they're not in your public profile.

Reply:
I think we can live without that unless that's somehow mandatory

Reply:
I can remove the `author` field if you want it to stay anonymous? (fyi John Doe is just a placeholder name)